src/hugo/content/en-us/implementations/aws/command/command2/_index.md

```diff
@@ -0,0 +1,721 @@
+- _Device_ is the IoT thing to be controlled
+- _Client Application_ is the remote logic that issues commands, update device state configuration and consumes device telemetry data
+- _Device Shadow_ is the replica which makes a device state available to applications and other services
+- _ShadowTopicPrefix_ is an abbreviated form of the topic, referred to `$aws/things/device1/shadow`(classic shadow topic)
+
+1. The _Device_ establishes an MQTT connection to the _AWS IoT Core_ endpoint and then subscribes to the reserved [MQTT shadow topics](https://docs.aws.amazon.com/iot/latest/developerguide/reserved-topics.html#reserved-topics-shadow) to get and update shadow event operations. Moreover, instead of subscribing individually to each reserved shadow topic, wildcard(_#_) could be used for generalized subscription like _`ShadowTopicPrefix/get/#`_ , _`ShadowTopicPrefix/update/#`_.
+
+    _`ShadowTopicPrefix/get/accepted`_   
+    _`ShadowTopicPrefix/get/rejected`_  
+    _`ShadowTopicPrefix/update/accepted`_    
+    _`ShadowTopicPrefix/update/rejected`_   
+    _`ShadowTopicPrefix/update/delta`_  
+
+2. After successful connect and topic subscription, the _Device_ publish request to _`ShadowTopicPrefix/get`_ topic and process the latest shadow file received on the _`ShadowTopicPrefix/get/accepted`_ topic. After processing configuration changes from _desired_ data in shadow file, _Device_ makes publish request to _`ShadowTopicPrefix/update`_ topic with device latest configuration as _reported_ data on shadow file.     
+3. Once the delta attributes are processed from the device shadow get request on device initial connect/reconnect, and if device optionally remain connected, then it can further receive delta changes on shadow file from the _Client Application_.   
+4. The _Client Application_ establishes an MQTT connection to the _AWS IoT Core_ endpoint, and then subscribes to the _`ShadowTopicPrefix/update/accepted`_, _`ShadowTopicPrefix/update/rejected`_ shadow topics to process configuration updates done on the device end. 
+5. For updating the device configuration, _Client Application_ publish message as _desired_ state on _`ShadowTopicPrefix/update`_ reserved topic. 
+
+```plantuml
+@startuml
+!define AWSPuml https://raw.githubusercontent.com/awslabs/aws-icons-for-plantuml/v7.0/dist
+!includeurl AWSPuml/AWSCommon.puml
+!includeurl AWSPuml/InternetOfThings/all.puml
+!includeurl AWSPuml/General/Client.puml
+
+'Comment out to use default PlantUML sequence formatting
+skinparam participant {
+    BackgroundColor AWS_BG_COLOR
+    BorderColor AWS_BORDER_COLOR
+}
+'Hide the bottom boxes
+hide footbox
+
+participant "<$IoTGeneric>\nDevice" as device
+participant "<$IoTShadow>\nShadow" as shadow
+participant "<$IoTCore>\nMQTT Broker" as broker
+participant "<$Client>\nApplication" as app
+
+== Device connect and subscribe ==
+device -> broker : connect(iot_endpoint)
+device -> broker : subscribe("$aws/things/device1/shadow/get/accepted")
```

Comment:
Do you need individual subscription or one to `$aws/things/device1/shadow/#`?

Reply:
I added for additional clarity, let me know if explicit mention will cause any issues.